src/Search/Input/QueryBuilder.php

```diff
@@ -80,6 +80,7 @@ public static function showGenericSearch($itemtype, array $params)
         $p['hide_controls']                 = false;
         $p['showmassiveactions']            = true;
         $p['extra_actions_templates']       = [];
+        $p['no_search']                     = $params['no_search'] ?? false;
```

Comment:
There is already a `no_search` parameter used within the search engine, so this naming could add to confusion. `hide_criteria` would probably be less confusing and match with the `hide_controls` parameter.

Reply:
Good call, done e6fce4698e.